site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -150,5 +150,11 @@ The `spec` field is the actual `GameServerAllocation` specification, and it is c
 - `metadata` is an optional list of custom labels and/or annotations that will be used to patch
   the game server's metadata in the moment of allocation. This can be used to tell the server necessary session data
 
-[gameserverselector]: {{% ref "/docs/Reference/agones_crd_api_reference.html#allocation.agones.dev/v1.GameServerSelector"  %}}
+Each `GameServerAllocation` also considers [namespaces][namespace] when allocating GameServers. Agones will only
```

Comment:
What about:

Each `GameServerAllocation` will allocate from a single namespace. The namespace can be specified....